2996_reflection/reflection.md

```diff
@@ -3098,6 +3098,15 @@ Prepend before paragraph 15 of [basic.def.odr]{.sref}:
 the program is ill-formed; a diagnostic is required only if the definable item is attached to a named module and a prior definition is reachable at the point where a later definition occurs. [...]
 :::
 
+Prefer the verb "denote" in bullet 15.5 to emphasize that ODR "looks through" aliases, and clarify that objects are not entities in bullet 15.5.2.
+
+::: std
+- [15.5]{.pnum} In each such definition, corresponding names, looked up according to [basic.lookup]{.sref}, shall [refer to]{.rm} [denote]{.addu} the same entity, after overload resolution ([over.match]{.sref}) and after matching of partial template specialization ([temp.over]{.sref}), except that a name can refer to
```

Comment:
I don't know how else to say this, but this paragraph sounds like William Shatner. There are too many commas and I think we're missing some words. And apparently that's just status quo, so... alrighty then.